dts/bindings/sensor/invensense,icm42605.yaml

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2020 TDK Invensense
+# SPDX-License-Identifier: Apache-2.0
+
+description: ICM-42605 motion tracking device
+
+# ICM-42605 is SPI.
+compatible: "invensense,icm42605"
+
+include: spi-device.yaml
+
+properties:
+    int-gpios:
+      type: phandle-array
+      required: true
+      description: |
+        The INT signal default configuration is active-high.  The
+        property value should ensure the flags properly describe the
+        signal that is presented to the driver.
+
+    cpha:
+      type: boolean
+      required: true
+      description: |
+        Set to indicate phase starts with asserted half-phase (CPHA=1).
+        For this driver using this property requires also using cpol.
+
+    cpol:
+      type: boolean
+      required: true
+      description: |
+        Set to indicate clock leading edge is falling (CPOL=1).
+        For this driver using this property requires also using cpha.
+
+    accel-hz:
+      type: int
+      required: false
+      default: 12
+      description: |
+        Default frequency of accelerometer.
+      enum:
+        - 1
+        - 3
+        - 6
+        - 12
+        - 25
+        - 50
+        - 100
+        - 200
+        - 500
+        - 1000
+        - 2000
+        - 4000
+        - 8000
+
+    gyro-hz:
+      type: int
+      required: false
+      default: 12
+      description: |
+        Default frequency of gyroscope.
+      enum:
+        - 12
+        - 25
+        - 50
+        - 100
+        - 200
+        - 500
+        - 1000
+        - 2000
+        - 4000
+        - 8000
+
+    accel-fs:
+      type: string
+      required: false
+      default: "16"
+      description: |
+        Default full scale of accelerometer. (Unit - g)
```

Comment:
Maybe add to comment that this maps to ACCEL_FS_SEL

Reply:
Need more description on yaml??

Reply:
Something like:
```
      description: |
        Default full scale of accelerometer. (Unit - g) [Maps to ACCEL_FS_SEL field in ACCEL_CONFIG0 setting]
```

Reply:
Do you meaning write more below information to description of yaml??

Full scale select for accelerometer UI interface output
000: ±16g (default)
001: ±8g
010: ±4g
011: ±2g

Reply:
I don't think that level of detail is necessary.  I was just saying having some reference to the datasheet.  I provided an example of the change to the `description` field that I was thinking about.

Reply:
Ah. understand. you means, just add "Maps to ACCEL_FS_SEL field in ACCEL_CONFIG0 setting" comments to description field.
Right?

Reply:
correct.

Reply:
@galak Added Maps comments.